README.md

```diff
@@ -28,11 +28,9 @@ You can either download the source from GitHub and compile yourself, or download
 Please see the [Download and Install](http://reactionmechanismgenerator.github.io/RMG-Py/users/rmg/installation/index.html) page for detailed instructions.
 
 ## Documentation
-- [RMG Documentation](http://ReactionMechanismGenerator.github.io/RMG-Py/) ([PDF version](https://github.com/ReactionMechanismGenerator/RMG-Py/raw/master/documentation/RMG-Py_and_CanTherm_Documentation.pdf))
-- [CanTherm Documentation](http://ReactionMechanismGenerator.github.io/RMG-Py/users/cantherm/) ([PDF version](https://github.com/ReactionMechanismGenerator/RMG-Py/raw/master/documentation/RMG-Py_and_CanTherm_Documentation.pdf))
-- [RMG API Reference](http://reactionmechanismgenerator.github.io/RMG-Py/reference/index.html) ([PDF version](https://github.com/ReactionMechanismGenerator/RMG-Py/raw/master/documentation/RMG-Py_API_Reference.pdf))
-
-Alternatively, PDF versions of all the documentation can also be found in the `documentation` folder of the source code.  
+- [RMG Documentation](http://ReactionMechanismGenerator.github.io/RMG-Py/users/rmg/index.html) ([PDF version](https://github.com/ReactionMechanismGenerator/RMG-Py/blob/master/documentation/RMG-Py_and_Arkane_Documentation.pdf))
+- [Arkane Documentation](http://ReactionMechanismGenerator.github.io/RMG-Py/users/arkane/index.html) ([PDF version](https://github.com/ReactionMechanismGenerator/RMG-Py/blob/master/documentation/RMG-Py_and_Arkane_Documentation.pdf))
+- [RMG API Reference](http://reactionmechanismgenerator.github.io/RMG-Py/reference/index.html) ([PDF version](https://github.com/ReactionMechanismGenerator/RMG-Py/blob/master/documentation/RMG-Py_API_Reference.pdf))
```

Comment:
I think you should keeps the pdf links to the files instead of the GitHub view (i.e. change `blob` back to `raw`).

Reply:
Thanks for catching this, done